### PR TITLE
fixes for writing logs to file and console

### PR DESCRIPTION
**Pull request by micahwoodard**
### Pull Request instructions:
- Please follow the [update protocol](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki/Software-Update-Procedures)
- Answer the questions below in detail. Your responses will be emailed to experimenters. 
- If the experimenters must do anything new, provide detailed step by step instructions on the [wiki](https://github.com/AllenNeuralDynamics/aind-behavior-blog/wiki)
- If computer maintainers need to manually update anything, provide detailed step by step instructions
- Use markdown syntax in order for your comments to be rendered reliably in the email: "1." instead of "1)", use four spaces for indents.
- If you use the keyword "skip email" in the title, it will skip the email updates
- Merges from "develop" into "production_testing" should use the keyword "production merge" in the title for reliable indexing of updates
- Merges from "production_testing" into "main" should use the keyword "update main"
  
### Describe changes:
- importing aind-slims-api messes up logging to file and console. Code now clears root log handlers to write both to file and console as before
### What issues or discussions does this update address?
- N/A
### Describe the expected change in behavior from the perspective of the experimenter
- None
### Describe any manual update steps for task computers
- None
### Was this update tested in 446/447?
- [ ] 446

### CODE REVIEW

**Comment by micahwoodard** [comment]
I'm not an expert on logging but this works locally on my machine to restore logs to the txt file and display in console if in settings

**Comment by alexpiet** [comment]
You should double check that this doesn't mess with the slims logging. I'm not an expert with logging either

**Comment by micahwoodard** [comment]
@alexpiet Okay tested on rig and seems to be writing files and logging to console correctly. It doesn't seem to mess with slims logging because the slims logger is just writing to sys.stderr, which is what the gui sets up later on if the settings call for it.

File: src/foraging_gui/Foraging.py [hunk 1]
```diff
@@ -4649,14 +4652,12 @@ def start_gui_log_file(box_number):
...
-    logging.basicConfig(
-        format=log_format,
-        level=logging.INFO,
-        datefmt=log_datefmt,
-        handlers=[
-            logging.FileHandler(logging_filename),
-        ]
-    )
+
+    log_formatter = logging.Formatter(fmt=log_format, datefmt=log_datefmt)
```
Comment (alexpiet) [inline src/foraging_gui/Foraging.py:4656]: You don't set the level as logging.INFO, is that the default? I would probably set it explicitly

Comment (alexpiet) [reply src/foraging_gui/Foraging.py:4656]: maybe you are just inheriting the level from the existing logger? In that case I would absolutely set the level

Comment (micahwoodard) [reply src/foraging_gui/Foraging.py:4656]: Oh yeah I forgot to set the level I'll add that in

**Review by alexpiet — APPROVED** [review]

**Pull request merged**